test/unit/config.spec.js

```diff
@@ -17,11 +17,11 @@ chai.should();
 const config = require('./../../lib/config');
 
 // This is the file we are testing
-describe('config', function() {
+describe('config', () => {
```

Comment:
We need to be careful with this, it can break Mocha, I've been trying to avoid it (hopefully I haven't committed any).

Reply:
Since tests pass, I don't think we need to be strict